spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/command/CommandHandler.java

```diff
@@ -7,7 +7,6 @@
  * @param <T> The concrete type of {@link Command}
  */
 public interface CommandHandler<T extends Command> {
```

Comment:
I know it was me who forgot in the first place, but could you copy the copyright from `CommandHandlerRegistry` over :)?

Reply:
Done